#!/usr/bin/env python
PACKAGE = "chicago"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("gear_rateo",    double_t,    0, "The gear rateo", 1/37,  1/35, 1/40)
gen.add("wheel_radius",    double_t,    1, "The wheel radius", 0.1575, 0.1575, 0.1575)
gen.add("real_baseline",    double_t,    2, "The real baseline", 0.583, 0.583, 0.583)
# size_enum = gen.enum([ gen.const("Small",      int_t, 0, "A small constant"),
#                        gen.const("Medium",     int_t, 1, "A medium constant"),
#                        gen.const("Large",      int_t, 2, "A large constant"),
#                        gen.const("ExtraLarge", int_t, 3, "An extra large constant")],
#                      "An enum to set size")

exit(gen.generate(PACKAGE, "velocity_computation", "parameters"))
